samples/test-runner/build.gradle.kts

```diff
@@ -74,8 +76,8 @@ instrumentation {
 
                 scheduling {
                     quota {
-                        retryCount = 1
-                        minimumSuccessCount = 1
+                        retryCount = 10
```

Comment:
is it ad-hoc?

Reply:
i just trying to run multiple emulators, but it still hungs from time to time